### PR TITLE
Search: Add plugin or theme's URL on wordpress.org

### LINKED ISSUE #94: Expose the w.org URL in plugin search results
When using `wp plugin search`, there's no way to quickly access to the plugin's w.org page to get more information about the plugin.

An extra field for the URL would be useful.

### PR DESCRIPTION
**Pull request by GaryJones**
After the plugin_api() or theme_api() call is made, update the results with the generated URL for wordpress.org, made up of a hard-coded domain name, the plural of the item type, and the item's slug.

Fixes #94.

### CODE REVIEW

File: src/Plugin_Command.php [hunk 1]
```diff
@@ -152,6 +152,7 @@ public function status( $args ) {
 	 *     **icons**: Plugin's Icon Image Link
 	 *     **active_installs**: Plugin's Number of Active Installs
 	 *     **contributors**: Plugin's List of Contributors
+	 *     **url**: Plugin's URL on wordpress.org
```
Comment (schlessera) [inline src/Plugin_Command.php]: CS: There's a trailing space here.

File: src/WP_CLI/CommandWithUpgrade.php [hunk 2]
```diff
@@ -613,6 +613,11 @@ protected function _search( $args, $assoc_args ) {
 
 		$items = $api->$plural;
 
+		// Add `url` for plugin or theme on wordpress.org.
+		foreach ( $items as $index => $item_object ) {
+			$item_object->url = 'https://wordpress.org/' . $plural . '/' . $item_object->slug . '/';
```
Comment (schlessera) [inline src/WP_CLI/CommandWithUpgrade.php]: I'd prefer this to use substitution:
```
$item_object->url = "https://wordpress.org/{$plural}/{$item_object->slug}/";
```

File: src/Plugin_Command.php (hunk 1 above)
Comment (GaryJones) [reply src/Plugin_Command.php]: Good catch. Fixed in the rebased 26862c0.

**Comment by schlessera** [comment]
This currently fails on WP trunk: https://travis-ci.org/wp-cli/extension-command/jobs/406130240#L665-L704

I assume the `plugins_api()` function has changed.

**Comment by GaryJones** [comment]
> This currently fails on WP trunk

Trying a workaround first.

**Comment by schlessera** [comment]
Thanks for the pull request, @GaryJones !

**Pull request merged**